docs/src/getting-started/hardware-interface.adoc

```diff
@@ -1 +1,165 @@
+:lang: en
+:toc:
+
+[[cha:hardware-interface]]
+= System Requirements(((System Requirements)))
+
+= Linuxcnc Hardware
+
+= Hardware to run the LinuxCNC software
+A computer is required to run LinuxCNC itself.
+
+Minimum computer requirements can be found in the LinuxCNC docs: http://linuxcnc.org/docs/stable/html/getting-started/system-requirements.html
+
+The most common is to use a x86 computer (standard Intel / AMD computer)
+
+ARM computers such as the Raspberry Pi or Orange Pi can be used
+
+= Hardware Interface to CNC machine
+There are multiple different ways to interface between LinuxCNC controller software, and CNC hardware (such as stepper / servo drivers, limit switches, inputs and outputs etc.)
+
+Interfaces include:
+- Parallel Port
+- Ethernet
+- Ethercat
+- PCI / PCIe
+- SPI (where the computer has a native SPI interface, such as Raspberry Pi)
+- USB (_not_ realtime interface)
+
+A mix of different interfaces can be used. For example, a combination of ethercat for servo drives, and parallel port for additional General Purpose Inputs / Outputs (GPIO)
+
+Some of these solutions are usable for all aspects of hardware interfacing, and some have specific roles (e.g. non-realtime GPIO, for an operator inferface).
+
+== Parallel Port
+Using onboard motherboard parallel port, or a PCI/PCIe parallel port card.
+
+== Parallel Port Software Interface
+Realtime (time critical) tasks such as step generation are done in software on the LinuxCNC host - this means the parallel port interface is much more sensitive to the LinuxCNC computer's latency.
+
+__Advantages:__
+- Low cost
+- Simple configuration
+
+__Disadvantages:__
+- Sensitive to the LinuxCNC computer's latency
+- Limited inputs/outputs 
+- Some PCI/PCIe parallel port cards do not work well or do not properly support the required EPP mode
```

Comment:
EPP is not a requirement for p-port step generation etc. It is only required to use the Mesa and Pico cards.